spec/models/ucas_matched_application_spec.rb

```diff
@@ -262,26 +276,36 @@
   end
 
   describe '#application_accepted_on_apply?' do
-    it 'returns true if application is accepted on Apply' do
-      ucas_matching_data =
-        { 'Scheme' => 'D',
-          'Course code' => course.code.to_s,
-          'Provider code' => course.provider.code.to_s,
-          'Apply candidate ID' => candidate1.id.to_s }
-      ucas_matching_application = UCASMatchedApplication.new(ucas_matching_data, recruitment_cycle_year)
+    describe 'accepted' do
```

Comment:
Could these be context blocks instead, for consistency? Also, could do something like the following and simplify the it block descriptions:

```
context 'when the application is accepted on Apply' do
  it 'returns true'
...
```